CMR/python/README.md

```diff
@@ -0,0 +1,49 @@
+# CMR Python Wrapper
+
+A wrapper for https://github.com/nasa/Common-Metadata-Repository/ in python
+
+## Assumptions
+
+1. Python 3.2 or better.
+2. Can access [cmr.earthdata.nasa.gov][cmr]
```

Comment:
nitpick: capitalize CMR

Reply:
That's a markdown link reference, it matches what is at the bottom. The text is what is in the first set.

Reply:
I am going to fix the one below that, urs vs edl

Reply:
leaving this one still open to gauge how strongly you feel about it.